bakerydemo/settings/base.py

```diff
@@ -66,18 +82,22 @@
     "taggit",
     "wagtailfontawesome",
     "debug_toolbar",
+    "django_extensions",
+    "simple_robots",
```

Comment:
Leaving this here also seems like a potential footgun - another dependency to install that beginners might not understand the use of.